### PR TITLE
Aftershock: Neurosoft Aeronautics CBM

### PR DESCRIPTION
**Pull request by John-Candlebury**
Grants you the pilot proficiency while installed.

#### Summary

SUMMARY: Mods "Aftershock: Aeronautics bionic grants you the pilot proficiency while installed."

#### Purpose of change

Allow wider access to flying machines within Aftershock.

#### Describe the solution

_Shoves chip into brain... You know how to pilot that now._

Uses the new bionic-proficiency mechanic recently added by anothersimulacrum.

### Testing

Install/uninstall bionic.

### CODE REVIEW

File: data/mods/Aftershock/player/bionics.json [hunk 1]
```diff
@@ -50,6 +50,15 @@
...
     "time": 200
   },
+  {
+    "id": "afs_bio_neurosoft_aeronautics",
+    "type": "bionic",
+    "name": { "str": "Neurosoft: Aeronautics" },
+    "description": "A brain implant that grants instinctual knowledge about the operation of flying machines.",
+    "occupied_bodyparts": [ [ "head", 2 ] ],
+    "learned_proficiencies": [ "prof_helicopter_pilot" ],
+    "enchantments": [ "cranial_explosion" ]
```
Comment (anothersimulacrum) [inline data/mods/Aftershock/player/bionics.json:60]: Quite the price...

Comment (John-Candlebury) [reply data/mods/Aftershock/player/bionics.json:60]: Ugh, thanks for noticing it.

**Comment by Night-Pryanik** [comment]
Could this be mainlined? In vanilla we have Close Quarters CBM which is quite close to the idea of this CBM.

**Comment by John-Candlebury** [comment]
Brought it up with some of the other main devs, the decision was to wait until the initial Exodii implementation before adding more neural-interface bionics into the game. I'll remember to bring this up with I-am-erk once he finishes the faction.

**Comment by NoahGoldFox** [comment]
This would be awesome~ and make helicopters more relevant

**Comment by Mister-Cultist** [comment]
LET'S GOOOOOOOO

**Pull request merged**